src/jquery.multiselect.js

```diff
@@ -419,15 +430,15 @@
             self._traverse(e.which, this);
             break;
           case 13: // enter
-          case 32: //space
```

Comment:
I would prefer you leave the key code comments in.

Reply:
I think this was likely an unintended accidental change on my part.   I generally prefer more comments, so removing them is likely a goof.   Sorry, I will fix.